### DISCUSSION TITLE
Local cypress run hangs

### DISCUSSION THREAD

**Discussion by GabRog** [cypress-io/cypress#15090] [Questions and Help]
Hello,
a problem occured for me - while running cypress test suite on mac locally using latest chrome execution hangs before first test case (first it in my only suite). Everything in section beforeEach is done, then everything hangs. 
I found out that also in this situation if last in beforeEach is assertion, it does not cause test timeout and failure, but just hangs. 
I am using cypress version 6.4.0.

My cypress.json:
```
{
  "baseUrl": "https://control.cloud-sqa-shared.akamai.com/",
  "browser": "chrome",
  "requestTimeout" : 30000,
  "numTestsKeptInMemory": 0,
  "responseTimeout" : 50000,
  "pageLoadTimeout": 100000,
  "defaultCommandTimeout": 15000,
  "chromeWebSecurity": false,
  "supportFile": "cypress/support/index.js",
  "screenshotsFolder": "cypress/reports/mochareports/screenshots",
  "videosFolder": "cypress/reports/mochareports/videos",
  "videoUploadOnPasses": false,
  "videoCompression": false,
  "video": false,
  "screenshotOnRunFailure": false,
  "reporter": "cypress-multi-reporters",
  "reporterOptions": {
    "reporterEnabled": "mochawesome",
    "mochawesomeReporterOptions": {
      "reportDir": "cypress/reports/",
      "quite": true,
      "overwrite": false,
      "html": false,
      "json": true
    }
  },
  "env": {
    "FAIL_FAST_ENABLED": false,
    "FAIL_FAST_PLUGIN": false
  },
  "viewportWidth": 1536,
  "viewportHeight": 960
}
```

**Comment by bahmutov** [comment]
Can you provide a reproducible example? Otherwise, you can follow the troubleshooting guide https://on.cypress.io/troubleshooting

Tip: remove the reporters to make sure the hanging is not reporter-related
